source/includes/api/_before_you_get_started.md

```diff
@@ -56,6 +56,31 @@ Below you can find a list of RW API resources that use the `application` field:
 * [Vocabulary](/index-rw.html#vocabulary-and-tags)
 * [Widgets](/index-rw.html#widget10)
 
+## Caching
+
+Some services in the RW API rely on caching as a way to improve the performance and efficiency of their endpoints. By relying on caching mechanisms, services can serve pre-computed responses for commonly requested endpoints, reducing the need to perform the computations needed to serve the same request several times. Caching also has the added benefits of improving performance and reducing service load, which leads to better scalability overall.
+
+The RW API has a system-wide HTTP cache that you may use to cache your requests. This cache is based on [Fastly](https://www.fastly.com/), so please refer to its documentation if you are looking for a specific detail on its behavior. HTTP caching is opt-in, meaning that by default no caching mechanism is applied to any request -  if you'd like your endpoints to benefit from caching, you need to explicitly implement it.
+
+### RW API caching considerations
+
+For most common use cases, you just need to keep in mind the following:
+
+* The default cache TTL is 3 days.
+* Only responses with successful response codes such as 200, 203, 204 or 30X are taken into consideration when caching.
+* Authorization endpoint responses are **never** cached.
```

Comment:
what do you mean by this? I'm assuming you mean requests with authentication cookies, right?

Reply:
I meant responses from auth/user-related endpoints - I felt like it was a bit on the extra-safe side to highlight that auth/user information is ever cached. I've rephrased so it's clearer, lmk what you think